local-cli/generator/templates/_buckconfig

```diff
@@ -0,0 +1,9 @@
+
+[android]
+  target = Google Inc.:Google APIs:23
+
+[maven_repositories]
+  central = https://repo1.maven.org/maven2
+
+[alias]
+  android-app = //android/app:app
```

Comment:
Up to you, I think `android/app` is pretty similar to `android-app` and might teach people about Buck targets. But if you like `android-app` better go for it.